ega-data-api-commons/pom.xml

```diff
@@ -47,6 +47,33 @@
             <groupId>org.projectlombok</groupId>
             <artifactId>lombok</artifactId>
         </dependency>
+
+        <!-- SamTools dependencies -->
+        <dependency>
+            <groupId>com.github.samtools</groupId>
+            <artifactId>htsjdk</artifactId>
+            <version>2.23.0</version>
+        </dependency>
+
+        <dependency>
+            <groupId>com.squareup.okhttp3</groupId>
+            <artifactId>okhttp</artifactId>
+            <version>3.9.1</version>
+        </dependency>
+
+        <!-- Test -->
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-test</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-core</artifactId>
```

Comment:
Isn't mockito-core provided by the [spring-boot-starter-test](https://github.com/spring-projects/spring-boot/blob/07fb4b065d9be6f6ade37e1878ef7a57ee10bf1f/spring-boot-project/spring-boot-starters/spring-boot-starter-test/build.gradle#L16) dependency?

Reply:
I just removed it as it was not need